packages/web-api/src/methods.ts

```diff
@@ -222,272 +488,8 @@ import type {
   WorkflowsStepCompletedResponse,
   WorkflowsStepFailedResponse,
   WorkflowsUpdateStepResponse,
-  AdminAppsRequestsCancelResponse,
-  BookmarksAddResponse,
-  BookmarksEditResponse,
-  BookmarksListResponse,
-  BookmarksRemoveResponse,
-  AdminConversationsConvertToPublicResponse,
-  AdminConversationsLookupResponse,
-  AdminRolesAddAssignmentsResponse,
-  AdminRolesListAssignmentsResponse,
-  AdminRolesRemoveAssignmentsResponse,
-  AdminAppsActivitiesListResponse,
-  AdminFunctionsListResponse,
-  AdminFunctionsPermissionsLookupResponse,
-  AdminFunctionsPermissionsSetResponse,
-  AdminWorkflowsSearchResponse,
-  AdminWorkflowsUnpublishResponse,
-  AdminWorkflowsCollaboratorsAddResponse,
-  AdminWorkflowsCollaboratorsRemoveResponse,
-  AdminWorkflowsPermissionsLookupResponse,
-  AdminAppsConfigLookupResponse,
-  AdminAppsConfigSetResponse,
 } from './types/response/index';
 // Request types
-import type {
```

Comment:
🗒️ Might be nice to also swap 🪓 the `request` and `response` comments for these moving types!